backend/layers/processing/process_download.py

```diff
@@ -75,37 +91,106 @@ def remove_prefix(string: str, prefix: str) -> str:
         else:
             return string
 
-    def process(self, dataset_id: DatasetVersionId, dataset_uri: str, artifact_bucket: str, sfn_task_token: str):
+    @staticmethod
+    def get_job_definion_name(dataset_vid: str) -> str:
+        if os.getenv("REMOTE_DEV_PREFIX"):
+            stack_name = os.environ["REMOTE_DEV_PREFIX"].replace("/", "")
+            prefix = f"{os.environ['DEPLOYMENT_STAGE']}-{stack_name}"
+        else:
+            prefix = f"{os.environ['DEPLOYMENT_STAGE']}"
+        job_definition_name = f"dp-{prefix}-ingest-process-{dataset_vid}"
```

Comment:
I'm thinking we can improve on this name. Perhaps replace `ingest-process` with `process-validate` instead?

Reply:
it is used in more that validate. It's sued for validte, cxg, and seurat